.cbmc-batch/jobs/aws_cryptosdk_serialize_frame/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright≈ Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Remove extra character.

Reply:
Good catch, not sure how it got there. Just one of the perils of working on a Macbook with its fancy character shortcuts!